frontend/src/components/LandingPage/Tools/ToolsStyle.css

```diff
@@ -0,0 +1,9 @@
+img {
```

Comment:
What do you think about renaming these to be something more specific? I think that if you leave them this way all img tags and all jumbotron classes will be affected by your changes.

Reply:
It will not work if use a different class name with reactstrap at least, so would I have to use img from bootstrap in order to fix it?